website/src/models/group.js

```diff
@@ -84,6 +84,31 @@ GroupSchema.pre('save', function(next){
   next();
 })
 
+GroupSchema.pre('remove', function(next) {
```

Comment:
I think I'll leave this as it's, because we have no clear way of handling errors in a post middleware